src/components/input-text/readme.md

```diff
@@ -0,0 +1,162 @@
+# calcite-input-text
+
+<!-- Auto Generated Below -->
+
+## Usage
+
+### Basic
+
+```html
+<calcite-input value="Entered value" placeholder="My placeholder"></calcite-input>
```

Comment:
☝️ all instances of `calcite-input` in this readme need renamed to `calcite-input-text`.

Reply:
those are autogenerated, a GH action will fix the readme

Reply:
@jcfranco storybook won't build when I delete the readme. I'll just delete the api ref table hopefully that does it 
![image](https://user-images.githubusercontent.com/10986395/180586620-21823c0d-545e-4a7b-a2d6-0e713c4cf55c.png)